packages/gatsby-plugin-i18n/src/gatsby-node.ts

```diff
@@ -28,12 +28,12 @@ function mustValidOptions(options: unknown): GatsbyI18nPluginOptions {
 }
 
 export const onCreateNode: GatsbyNode['onCreateNode'] = async ({
-  node,
-  actions,
-  loadNodeContent,
-  createNodeId,
-  createContentDigest,
-}, options) => {
+                                                                 node,
+                                                                 actions,
+                                                                 loadNodeContent,
+                                                                 createNodeId,
+                                                                 createContentDigest,
+                                                               }, options) => {
```

Comment:
```suggestion
  node,
  actions,
  loadNodeContent,
  createNodeId,
  createContentDigest,
}, options) => {
```
헉 indent가 어마무시하게 생겼읍니다